gamestonk_terminal/portfolio/port_controller.py

```diff
@@ -0,0 +1,161 @@
+__docformat__ = "numpy"
+
+import argparse
+from typing import List
+from prompt_toolkit.completion import NestedCompleter
+from gamestonk_terminal import feature_flags as gtff
+from gamestonk_terminal.helper_funcs import get_flair
+from gamestonk_terminal.menu import session
+from gamestonk_terminal.portfolio import rh_api
+
+
+class PortfolioController:
+
+    CHOICES = [
+        "help",
+        "q",
+        "quit",
+        "login",
+        "hold",
+        "rhhist",
+    ]
+
+    BROKERS = [
+        "rh",
+    ]
+
+    def __init__(self):
+        self.port_parser = argparse.ArgumentParser(add_help=False, prog="port")
+        self.port_parser.add_argument("cmd", choices=self.CHOICES)
+        self.brokers_list = set()
+
+    @staticmethod
+    def print_help(broker_list):
+        if not broker_list:
+            print_broke = " None"
+        if broker_list:
+            print_broke = " "
+            for broker in broker_list:
+                print_broke += broker + " "
+
+        """ Print help """
+        print("\nPortfolio:")
+        print("   help          show this menu again")
+        print(
+            "   q             quit this menu, and shows back to main menu, logs out of brokers"
+        )
+        print("   quit          quit to abandon program, logs out of brokers")
+        print("   login         login to your brokers")
+        print("\nCurrent Brokers :" + print_broke)
```

Comment:
You can do something like this in one line:
```
print(f"\nCurrent Brokers : {('', ', '.join(broker_list))[bool(broker_list)]}")
```

So, `', '.join(['a', 'b', 'c']) ` would resulting in the string `a, b, c`. 

Then you use the ternary operation
```
('A', 'B')[False] = 'A'
('A', 'B')[True] = 'B'
```

Reply:
Love the python golf!

Reply:
You live and you learn :D